### ISSUE TITLE
Batching for speed.

### ISSUE THREAD

**Issue by wj210** [shmsw25/FActScore#45]
Hi, i would like to ask if it has been tried out with llama that  batch inference works?

i followed this https://huggingface.co/docs/transformers/llm_tutorial#wrong-padding-side , where they pass both the input_ids and attention_mask to the model, but i got 'nan' values. If i only passed in the input_ids, it is fine, however I'm not sure if not passing in the attention mask will have any effects on the final output.

Also, there seems to be a bug in 
https://github.com/shmsw25/FActScore/blob/c6e150ffc0f14162c7f603744f19d63c0e5b5469/factscore/lm.py#L27

where `" True or False?\nAnswer:"` is not detected since https://github.com/shmsw25/FActScore/blob/c6e150ffc0f14162c7f603744f19d63c0e5b5469/factscore/factscorer.py#L223 ends with a different output, hence the model generated length is 128 instead of 1. This will waste cost and if in the case of gpt3.5 being used, there may be both 'true' and 'false' in the 128 tokens leading to wrong decisions.
